### PR TITLE
run the frequently failing test-services first

### LINKED ISSUE #7918: Run test-services first in Linux/Windows JS CI tests
While working on #7864 over the weekend, the Linux/Windows JS CI tests each had at least half a dozen spurious failures. Every single time the failure occurred during the run of the test-services package. The test-whatever packages are run in alphabetical order, so test-services isn't run until near the end of the tests, at around the ~30-35 min mark.

I think it would make the CI slightly less painful to deal with if test-services ran first. The tests would still fail the same, of course, but at least that way we wouldn't have to wait 30 minutes just to see the test die. Is this a sensible proposal?

### PR DESCRIPTION
**Pull request by telamonian**
resolves #7918

### CODE REVIEW

File: scripts/appveyor.cmd [hunk 1]
```diff
@@ -25,6 +25,7 @@ IF "%NAME%"=="python" (
     if !errorlevel! neq 0 exit /b !errorlevel!
     jlpm run build:test
     if !errorlevel! neq 0 exit /b !errorlevel!
+    setx FORCE_COLOR 1 && jlpm coverage:services --loglevel success
```
Comment (blink1073) [inline scripts/appveyor.cmd]: I say we allow services to run twice since we inevitably just end up kicking it

Comment (telamonian) [reply scripts/appveyor.cmd]: I think we already do run services twice (if it fails the first time) here:

https://github.com/jupyterlab/jupyterlab/blob/c4ec346ddf38887d876ddcdca8b5bc747967cbbd/tests/test-services/package.json#L8

I'm not actually sure if that's been working as intended, though

Comment (telamonian) [reply scripts/appveyor.cmd]: Hmm, actually, there is one problem with how we're doubling up the run test-services cmd above: it messes up how command line arguments get passed down. For example, if I run:

```
setx FORCE_COLOR 1 && jlpm coverage:services --detectOpenHandles
```

it resolves to:

```
python run.py --coverage || python run.py --coverage --detectOpenHandles
```

I'll play around with this

**Comment by telamonian** [comment]
In other horrifying news, reordering the tests like this seems to shave about 5 min off the runtime of the "foo JS" suites

**Comment by blink1073** [comment]
Can you add an `|| jlpm run test:examples` to the usage test as well?   That is the other one one I've been having to restart for lately.

**Comment by telamonian** [comment]
The coverage stuff seems to be hopelessly broken. I have a sneaking suspicion that the problem has something to do with how our project is structured; basically, the issue may be that the `rootDir` (as defined in various `tsconfig.json` and `jest.config.js` files) in the test package doesn't match that of the real package. I was able to "fix" the errors during coverage collection by futzing with `rootDir`, but I was still unable to get it do anything useful: no errors, but coverage was measured at 0% for all files.

Is there a point in the past when it was known to be working correctly?

**Comment by blink1073** [comment]
See discussion about coverage here: https://github.com/jupyterlab/jupyterlab/issues/6415

**Comment by telamonian** [comment]
https://github.com/jupyterlab/jupyterlab/issues/6415#issuecomment-496164789

That `TypeError:... Emit skipped` thing was driving me nuts. How did you de-obfuscate it?

**Comment by blink1073** [comment]
I don't remember unfortunately...

**Comment by blink1073** [comment]
Closing since our tests and CI have been completely refactored

**Pull request closed without merging**